src/meta_schedule/schedule_rule/multi_level_tiling_with_intrin.cc

```diff
@@ -85,21 +101,23 @@ class MultiLevelTilingWithIntrinNode : public MultiLevelTilingNode {
 
  public:
   /*! \brief The name of a tensor intrinsic. */
-  String intrin_name;
+  Array<String> intrin_name;
```

Comment:
if the field type is still be changed, I recommend to rename it to `intrin_names` for the sake of clarity

Reply:
Outdated.